kitsune/wiki/templates/wiki/review_revision.html

```diff
@@ -20,6 +20,15 @@
         <p>
           {{ _('This revision was already reviewed by {user} on {date}.')|fe(user=revision.reviewer, date=datetimeformat(revision.reviewed, format='longdatetime')) }}
         </p>
+      {% elif current_revision_id > revision.id %}
+        <p>
+          {{ _('A newer revision has already been reviewed.') }}
+          {% if latest_revision_id > current_revision_id %}
+          <br>
+          {{ _('But there is another latest revision which is waiting for review.') }}
```

Comment:
This line, and the one above it, need indented, since they are inside the if block from line 26.